pkgs/by-name/ed/ed/package.nix

```diff
@@ -1,9 +1,40 @@
-{ lib
-, fetchurl
+{
+  lib,
+  callPackage,
+  lzip,
+  runtimeShellPackage,
+  stdenv,
+  # Configurable options
+  sources ? callPackage ./sources.nix { },
```

Comment:
What is the point of this compared to `overideAttrs` and using `finalAttrs` pattern?

Reply:
I was writing an expression that would build both edUnstable and ed. To facilitate this, I have factored sources acquisition in a separate file.

Reply:
I thought that `edUnstable` was removed.

Reply:
Yes, I removed it because it does not exist anymore  (I verified it today). So I removed the detached sources.nix.